Gopkg.toml

```diff
@@ -0,0 +1,15 @@
+[[constraint]]
+  name = "github.com/go-sql-driver/mysql"
+  version = "1.3.0"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/lib/pq"
+
+[[constraint]]
+  name = "github.com/mattn/go-sqlite3"
+  version = "1.2.0"
```

Comment:
Can you make all these constraints start with `^` character, so we can update easily?

```diff
- version = "1.2.0"
+ version = "^1.2.0"
```